misc/userscripts/password_fill

```diff
@@ -236,7 +236,7 @@ pass_backend() {
         if ((match_line)) ; then
             # add entries with matching URL-tag
             while read -r -d "" passfile ; do
-                if $GPG "${GPG_OPTS}" -d "$passfile" \
+                if $GPG "${GPG_OPTS[@]}" -d "$passfile" \
```

Comment:
It was the intent to print all `GPG_OPTS`, i.e. yes, it's a mistake. Thanks for finding & fixing that!